translating_strings/Italiano.txt

```diff
@@ -5,7 +5,7 @@ English: Pseudo Legend
 Translation: Pseudo Leggendario
 
 English: Box Legendary
-Translation: Scatola leggendaria
```

Comment:
"Scatola leggendaria" reads as "legendary box". No idea what the proper term for this is, but at least "Leggendario di scatola" reads as "legendary of the box" which is still an improvement.